command/sub8_missions/missions/align_channel.py

```diff
@@ -1,63 +1,48 @@
 from __future__ import division
 
+import genpy
 import tf
 from twisted.internet import defer
 from txros import util
 from sub8 import Searcher
-from sub8_ros_tools import pose_to_numpy
+from sub8_ros_tools import pose_to_numpy, rosmsg_to_numpy
 import numpy as np
 
-
 SEARCH_DEPTH = .65
-SPEED = .3
-
-def catch_error(failure):
-    print failure.printTraceback()
-
+TIMEOUT_SECONDS = 60
 @util.cancellableInlineCallbacks
 def run(sub):
-    # Go to max height to see as much as we can.
-    yield sub.move.depth(SEARCH_DEPTH).zero_roll_and_pitch().go(speed=SPEED)
-    yield sub._node_handle.sleep(1.0)
-
-    # Do a little jig - change this for the pool.
-    pattern = [sub.move.right(1), sub.move.forward(1), sub.move.left(1), sub.move.backward(1),
-               sub.move.right(2), sub.move.forward(2), sub.move.left(2), sub.move.backward(2)]
-    s = Searcher(sub, sub.channel_marker.get_pose, pattern)
+    print "MARKER MISSION - STARTING"
```

Comment:
Yeah, I didn't like all the mixed references to marker/orange_path/alignment/etc last year